pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -258,435 +259,447 @@ private BrokerResponseNative handleRequest(long requestId, String query,
       throws Exception {
     LOGGER.debug("SQL query for request {}: {}", requestId, query);
 
-    long compilationStartTimeNs;
-    PinotQuery pinotQuery;
+    Tracing.ThreadAccountantOps.setupRunner(String.valueOf(requestId));
```

Comment:
Should we add a comment stating that this must not be moved further below interspersed into the code ?

Reply:
Good point, added